Hardcore.lua

```diff
@@ -263,6 +263,17 @@ function Hardcore:PLAYER_LEVEL_UP(...)
 	--take screenshot (got this idea from DingPics addon)
 	-- wait a bit so the yellow animation appears
 	C_Timer.After(PICTURE_DELAY, Screenshot)
+
+	-- send a message to the guild if the player's level is divisible by 10
+	local landmarkLevel = (level % 10) == 0
+	if (landmarkLevel) then
+		local playerName = UnitName("player")
+		local localizedClass = UnitClass("player")
```

Comment:
This is kind of a nit, but if someone is running e.g. the German or French client, the message sent to guild chat would be a bit weird having a non-English word in an English sentence.

According to the docs `UnitClass` also returns the english class name:
https://wowwiki-archive.fandom.com/wiki/API_UnitClass

I would change it to:

```
local _, englishClass = UnitClass("player")
```

Reply:
Done, cheers

Reply:
This causes allcaps, weirdly.

We use localised classes in other messages, so I'm going to revert this.